otwcode/otwarchive
# AO3-6518 Spam check runs on abuse reports if the email you enter has different capitalization than the email on your account

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6518

## Purpose

Previously, the spam checker was flagging the report as spam if the casing of the emails for the account and submission did not match. We fixed this by downcasing both values before comparing.

## Testing Instructions
1. Log in
2. Follow “Policy Questions & Abuse Reports” in the site footer
3. Change the “Your name (optional)” field to say “akismet-guaranteed-spam,” which will guarantee your report is flagged as spam if the spam checker runs on it
4. Change the capitalization of the email in the “Your email (required)” field
5. Fill in rest of form
6. Submit

Form should submit successfully.

## Credit

calm (they/them)
